tests/unit_test.py

```diff
@@ -570,3 +572,15 @@ def test_covert_to_unit_should_fail(src, dest):
     dest_unit = process_sympy(dest, parse_as_unit=True)
     with pytest.raises(Exception):
         convert_to(src_unit, dest_unit)
+
+
+def test_ensure_aliases_are_pickled():
+    tests_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '.'))
+    unit_aliases_file = open(f'{tests_dir}/../src/latex2sympy/units/unit_aliases/unit_aliases.py', 'r')
+    h = hashlib.new('sha256')
+    h.update(unit_aliases_file.read().encode('utf-8'))
+    unit_aliases_file.close()
+    file_hash = h.hexdigest()
+    assert file_hash == 'b35d020318717bcf65105f83e5a4d1b39785c7a8db7da6f8a4241d2722d2f5af', \
```

Comment:
@tragessere @jkdufair yeah these hash checks do still feel kind of "extra", and like you said don't cover all possible files that can affect the output, and don't check the output files themselves

personally, i was fine without them, since if you are adding a new parser rule or a new unit, you're gonna be adding unit tests to confirm those rules/units now work

Reply:
I like the idea of  a pre-commit hook. I hate to leave anything to "We better remember". Even if we add new unit tests etc for new rules/units we still have to remember to pickle stuff. I guess if the unit tests are using the pickle file they will fail. So maybe it's fine. But we might still thrash around wondering why the tests are failing. Especially if it's not one of us that worked on this bit.

Alternatively, we could hash the pickle file itself and have the unit test check that. @tragessere - you don't have to manually has the file - the test gives you the new hash and you just have to paste it in.